### ISSUE TITLE
Add new programs of c++ and python for Hacktoberfest 2021

### ISSUE THREAD

**Issue by Apurvgeek** [Apurvgeek/programs-in-cpp-and-python#15]
Contribute to the society

**Comment by Foo-Maker** [comment]
Hey.
I have writen a little program to check the status from a website.
(at the moment are the services from facebook down)

is this ok?

**Comment by shaurya5** [comment]
I can add more programs. Please assign me this issue.

**Comment by Apurvgeek** [comment]
> 
> 
> I can add more programs. Please assign me this issue.

Didn't get the PR, did you commit it with the main branch??

**Comment by Apurvgeek** [comment]
> 
> 
> Hey. I have writen a little program to check the status from a website. (at the moment are the services from facebook down)
> 
> is this ok?

Yup it will be fine, go ahead and create a pull request to the main branch

**Comment by shaurya5** [comment]
> > I can add more programs. Please assign me this issue.
> 
> Didn't get the PR, did you commit it with the main branch??

I didn't commit yet, will commit in some time

**Comment by Foo-Maker** [comment]
> > Hey. I have writen a little program to check the status from a website. (at the moment are the services from facebook down)
> > is this ok?
> 
> Yup it will be fine, go ahead and create a pull request to the main branch

Thank you. 

My PR #17

**Comment by plavani0208** [comment]
Can i add a password generator code?

**Comment by Apurvgeek** [comment]
Sure go ahead 🙂

**Comment by Yash-RajSingh** [comment]
Hey can i add programs like a graphical periodic table and calculatior in C++? If yes please assign this issue to me, i would really love to contribute.
